src/main/java/com/troubleshooting/model/ConjurLogModel.java

```diff
@@ -0,0 +1,28 @@
+package com.troubleshooting.model;
+
+// Data object for a Conjur log
+public class ConjurLogModel implements ILogModel {
+    private String logLevel;
+    private String message;
+
+    public String getLogLevel() {
```

Comment:
IMHO any public method should be documented.

Reply:
do you suggest it they be private?  and if so why?

Reply:
"If you like it than you should have out a ring on it"
If it's public it should be documented.